app/models/employer_group_size_rating_factor_set.rb

```diff
@@ -8,6 +8,6 @@ def self.value_for(carrier_profile_id, year, val)
 
   def lookup(val)
     lookup_key = (val > max_integer_factor_key) ? max_integer_factor_key : val
-    super(lookup_key.to_s)
+    super(lookup_key.to_f.to_s)
```

Comment:
I don't understand why you added this.

What factor_key is 1.0? This caused the tests written in the commit to fail - did you run them after you made the change?